src/models/AppUser.js

```diff
@@ -51,9 +51,10 @@ class AppUser extends Resource {
   }
   /**
    * @param {string} appId
+   * @param {object} queryParameters
    */
-  delete(appId) {
-    return this.client.deleteApplicationUser(appId, this.id);
+  delete(appId, queryParameters) {
```

Comment:
Are these required or optional?  I'm assuming optional, otherwise this would be a major rev

Reply:
It is optional, and having this does not seem to break any tests